[2024-06-21T07:36:01.787+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-21T07:36:01.809+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-21T07:35:00+00:00 [queued]>
[2024-06-21T07:36:01.818+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-21T07:35:00+00:00 [queued]>
[2024-06-21T07:36:01.819+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-21T07:36:01.834+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-21 07:35:00+00:00
[2024-06-21T07:36:01.839+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3057) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-21T07:36:01.840+0000] {standard_task_runner.py:63} INFO - Started process 3060 to run task
[2024-06-21T07:36:01.840+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_apis_final', 'check_previous_run_status', 'scheduled__2024-06-21T07:35:00+00:00', '--job-id', '3000', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting_apis.py', '--cfg-path', '/tmp/tmp3_7qw_de']
[2024-06-21T07:36:01.841+0000] {standard_task_runner.py:91} INFO - Job 3000: Subtask check_previous_run_status
[2024-06-21T07:36:01.888+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-21T07:35:00+00:00 [running]> on host e66744caf823
[2024-06-21T07:36:01.989+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_apis_final' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-21T07:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-21T07:35:00+00:00'
[2024-06-21T07:36:01.989+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-21T07:36:02.002+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-21T07:36:02.002+0000] {python.py:237} INFO - Done. Returned value was: skip_run
[2024-06-21T07:36:02.003+0000] {branch.py:36} INFO - Branch into skip_run
[2024-06-21T07:36:02.003+0000] {skipmixin.py:178} INFO - Following branch skip_run
[2024-06-21T07:36:02.018+0000] {skipmixin.py:246} INFO - Skipping tasks [('start_run', -1)]
[2024-06-21T07:36:02.042+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-21T07:36:02.059+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_apis_final, task_id=check_previous_run_status, run_id=scheduled__2024-06-21T07:35:00+00:00, execution_date=20240621T073500, start_date=20240621T073601, end_date=20240621T073602
[2024-06-21T07:36:02.093+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-21T07:36:02.147+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-21T07:36:02.151+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
